engine/src/main/java/org/terasology/world/generation/WorldImpl.java

```diff
@@ -67,9 +67,6 @@ public void rasterizeChunk(CoreChunk chunk, EntityBuffer buffer, float scale) {
         for (WorldRasterizer rasterizer : scalableWorldRasterizers) {
             ((ScalableWorldRasterizer) rasterizer).generateChunk(chunk, chunkRegion, scale);
         }
-        for (EntityProvider entityProvider : entityProviders) {
-            entityProvider.process(chunkRegion, buffer);
```

Comment:
You can remove `EntityBuffer buffer` parameter, because it not used.
This object used only for entity generation during world generation.